src/main/java/it/gov/pagopa/afm/calculator/entity/Bundle.java

```diff
@@ -44,8 +43,7 @@ public class Bundle {
     @Enumerated(EnumType.STRING)
     private PaymentMethod paymentMethod;
 
-    @Enumerated(EnumType.STRING)
-    private Touchpoint touchpoint;
+    private String idTouchpoint;
```

Comment:
the entity differs from that defined on [marketplace](https://github.com/pagopa/pagopa-afm-marketplace-be/pull/45/files#diff-950025bb2f746939272e5493b545101569b498d791b84d67d8e351b127506032R47)